cmake/projects/spdlog/hunter.cmake

```diff
@@ -137,6 +137,17 @@ hunter_add_version(
     SHA1          f3219fb548667857f95d20714a2c85a4ae41049b
 )
 
+hunter_add_version(
+    PACKAGE_NAME
+    spdlog
+    VERSION
+    "1.12.0-p0"
+    URL
+    "https://github.com/JamesEggleton/spdlog/archive/refs/tags/v1.12.0-p0.tar.gz"
```

Comment:
This URL will need updating once related PR https://github.com/cpp-pm/spdlog/pulls has been approved.

Reply:
done: https://github.com/cpp-pm/spdlog/releases/tag/v1.12.0-p0

please update and rebase on latest master, such that this PR only contains changes for `spdlog` package